services/weblate/weblate-user-statistics.service.js

```diff
@@ -0,0 +1,92 @@
+'use strict'
+
```

Comment:
Won't rehash everything but several of the same feedback items for this file as well